.github/workflows/rust.yml

```diff
@@ -22,6 +22,6 @@ jobs:
       labels: bottlerocket_ubuntu-latest_16-core
     steps:
       - uses: actions/checkout@v3
-      - run: cargo install --locked cargo-deny
-      - run: cargo install --locked cargo-make@0.37.12
```

Comment:
I think we originally locked to `0.37.12` for `cargo-make` due to a tendency to introduce breaking changes upstream. Might be worth continuing to require a specific version.

Reply:
Actually in this case I added this version lock before realizing that we needed to bump nightly. The newer patch of cargo-make would pull in the serde version that was incompatible with the old nightly

Reply:
Ah, I just misremembered. Forget my feedback then! 🚀